scripts/krakenuniq

```diff
@@ -230,7 +230,7 @@ if ($paired) {
 my $cmd = $use_exact_counting? $CLASSIFY_EXACT : $CLASSIFY;
 print STDERR "$cmd @flags\n";
 if (defined $report_file) {
-    my $mycmd = qx/ps -o args= $$/;
+    my $mycmd = 'krakenuniq ' . join(' ', @ARGS);
```

Comment:
It seems the `krakenuniq` script uses `@ARGV` and not `@ARGS`.
```suggestion
    my $mycmd = 'krakenuniq ' . join(' ', @ARGV);
```
However, I don't think this is enough.

For reference an example of the change in the report output looks like this, original first (first two lines only):
```
# KrakenUniq v1.0.0 DATE:2022-09-29T12:12:43Z DB:/ceph/db/krakenuniq/minikraken_20171013_4GB DB_SIZE:3758097436 WD:/ceph/home/boulund/work/stag-uniq/.snakemake/shadow/tmp_1fp1y3a
# CL:perl /ceph/home/boulund/work/stag-uniq/.snakemake/conda/207ba047bd7ea05bc7d1265affc0c46c/bin/krakenuniq --db /ceph/db/krakenuniq/minikraken_20171013_4GB --threads 4 --output output_dir/krakenuniq/test1.kraken.gz --report-file output_dir/krakenuniq/test1.kreport --preload-size 2G --paired output_dir/host_removal/test1_1.fq.gz output_dir/host_removal/test1_2.fq.gz
```
then the result after the suggested changes (first two lines only):
```
# KrakenUniq v1.0.0 DATE:2022-09-30T07:34:35Z DB:/ceph/db/krakenuniq/minikraken_20171013_4GB DB_SIZE:3758097436 WD:/ceph/home/boulund/work/stag-uniq/.snakemake/shadow/tmp641_zm3v
# CL:krakenuniq output_dir/krakenuniq/merged.fa
```

I don't think the suggested change is the solution since it don't seem to capture the entire command line. I'm guessing this is due to Getopt chomping the commandline so only the positional arguments remains in `@ARGV`.